assets/css/components/_notifications.scss

```diff
@@ -97,3 +97,12 @@
     }
   }
 }
+
+// first-login-modal-eth
+.modal-body {
+  img.width-15 {
+    @media (max-width: 991px) {
+      height: 150px !important;
+    }
+  }
+}
```

Comment:
Instead of adding another rule can we add `.width-20-mobile.width-20-tablet` classes to the img in first-login-modal-eth please :)?